src/nfa/nfa-inclusion.cc

```diff
@@ -152,25 +188,44 @@ bool Mata::Nfa::Algorithms::is_incl_antichains(
 
 				if (is_subsumed) { continue; }
 
-				// prune data structures and insert succ inside
-				for (std::list<ProdStateType>* ds : {&processed, &worklist}) {
-					auto it = ds->begin();
-					while (it != ds->end()) {
-						if (subsumes(succ, *it)) {
-							auto to_remove = it;
-							++it;
-							ds->erase(to_remove);
-						} else {
-							++it;
-						}
-					}
-
-					// TODO: set pushing strategy
-					ds->push_back(succ);
-				}
+                // prune data structures and insert succ inside
```

Comment:
I would not keep commented code in here. Delete or maybe add guard? `#if ON_MACOS ... #endif`?

Reply:
should be removed.